### PR TITLE
Sanitize and remove NCATS references

### PR DESCRIPTION
**Pull request by dav3r**
#  #

## 🗣 Description ##

This PR does the following:
* Sanitizes all public IPs in test data by replacing them with private IPs
* Adds the complete set of sanitized email tokens to the test data
* Removes all NCATS (our old team name) references and replaces them with `cisagov`
* Cleans up the README a bit to conform to our standards

## 💭 Motivation and context ##

I needed to build a fresh [Docker image for `pca-reports`](https://hub.docker.com/r/cisagov/pca-reports) (which depends on the Docker image from this repo) and I noticed that this repo needed a bit of cleanup, so I went ahead and took care of it.

## 🧪 Testing ##

I ran `pytest` and verified that all tests are passing.

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
* [x] Tests have been added and/or modified to cover the changes in this PR.
* [x] All new and existing tests pass.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,31 +47,31 @@ Whenever an aliased PCA command is executed, it will use the current working dir
...
+### Advanced configuration
 
-By default, the container will look for your PCA configuration in `/etc/pca`. This location can be changed by setting the `PCA_CONF_DIR` environment variable to point to your PCA configuration directory. The commands will also attempt to run using the `ncats/pca-core` image. A different image can be used by setting the `PCA_CORE_IMAGE` environment variable to the image name.
+By default, the container will look for your PCA configuration in `/etc/pca`. This location can be changed by setting the `PCA_CONF_DIR` environment variable to point to your PCA configuration directory. The commands will also attempt to run using the `cisagov/pca-core` image. A different image can be used by setting the `PCA_CORE_IMAGE` environment variable to the image name.
 
 Example:
 
 ```
 export PCA_CONF_DIR=/private/etc/pca
-export PCA_CORE_IMAGE=dhub.ncats.cyber.dhs.gov:5001/pca-core
```
Comment (jsf9k) [inline README.md:58]: Antediluvian!

File: pca/test/test_XMLData/tokens.xml [hunk 2]
```diff
@@ -1,8 +1,307 @@
 <tokens>
-	<!-- Test entries likely need to be added before testing will work -->
 	<entry>
-			<token>0123456789ab</token>
+			<token>04ea957ec11c</token>
 			<email>test1@example.org</email>
-			<cid>2800000a-500e-4008-a00e-600000000007</cid>
+			<cid>28fc34fa-51be-4518-a7be-656cbdf01ae7</cid>
```
Comment (jsf9k) [inline pca/test/test_XMLData/tokens.xml:5]: Why did you change the token and cid values?  I just want to make sure there isn't some leakage here.

**Review by jsf9k — APPROVED** [review]
Approved, but I had one question - probably because I am utterly ignorant of this ancient repository.

File: README.md (hunk 1 above)
Comment (dav3r) [reply README.md:58]: Yeah- really getting rid of some old sauce here! 🌊

File: pca/test/test_XMLData/tokens.xml (hunk 2 above)
Comment (dav3r) [reply pca/test/test_XMLData/tokens.xml:5]: The token and cid were unnecessarily sanitized when they were moved from the private repository to this public one.  I changed them back so that they align with the values in the rest of the test data. There is no sensitive info here.

**Comment by dav3r** [comment]
BTW, I added @teal-e and @BenBreaksThings as reviewers, but mainly for their awareness of this PR.

File: pca/test/test_XMLData/tokens.xml (hunk 2 above)
Comment (hillaryj) [reply pca/test/test_XMLData/tokens.xml:5]: Coolio - I'd sanitized them just-in-case without the context of whether they were sensitive or not. Glad to hear they aren't!

**Review by hillaryj — APPROVED** [review]
😍
